packages/flutter_adaptive_scaffold/lib/src/breakpoints.dart

```diff
@@ -84,6 +187,14 @@ class WidthPlatformBreakpoint extends Breakpoint {
   /// upper bound.
   final double? end;
 
+  /// The beginning height dp value. If left null then the [Breakpoint] will have
+  /// no lower bound.
+  final double? top;
```

Comment:
@gspencergoog what do you think of this name? The others are begin, end so maybe this should be beginHeight, endHeight?

Reply:
Yes, that makes more sense to have it be `beginHeight` and `endHeight`.  begin/end/top/bottom makes me think of rectangle boundaries in a directional widget.

Reply:
Yes, it makes more sense to me to have it be `beginHeight` and `endHeight`.  begin/end/top/bottom makes me think of rectangle boundaries in a directional class (e.g. `EdgeInsetsDirectional`, which uses start instead of begin, but still). Of course, that means you'll want to rename `begin` and `end` to `beginWidth` /`endWidth` too.

Reply:
@gspencergoog I've implemented this now.